### PR TITLE
[release] adding 5.22.2 release

### PR DESCRIPTION
**Pull request by truthbk**

### CODE REVIEW

File: release.json [hunk 1]
```diff
@@ -10,6 +10,17 @@
 		"OMNIBUS_RUBY_BRANCH": "datadog-5.5.0",
 		"OMNIBUS_SOFTWARE_BRANCH": "master"
 	},
+	"5.22.2": {
+		"AGENT_VERSION": "5.22.2",
+		"AGENT_BRANCH": "5.22.2",
+		"AGENT6_BRANCH": "6.0.1",
+		"INTEGRATIONS_CORE_BRANCH": "5.22.2",
+		"TRACE_AGENT_BRANCH": "5.22.0",
```
Comment (LotharSee) [inline release.json]: Let's move to `6.0.1` for the `trace-agent`.
Good to you?
I don't know how we should tag/version it while we still have A5 and A6.

Comment (truthbk) [reply release.json]: Normally A5 and A6 releases should be chronologically aligned, so I think continuing with `6.x` tags should be fine. Also if for once we're not strict about semver it's not the end of the world.

Comment (LotharSee) [reply release.json]: Okidoki so let's go with this one 👍

File: release.json [hunk 2]
```diff
@@ -10,6 +10,17 @@
...
 		"OMNIBUS_SOFTWARE_BRANCH": "master"
 	},
+	"5.22.2": {
+		"AGENT_VERSION": "5.22.2",
+		"AGENT_BRANCH": "5.22.2",
+		"AGENT6_BRANCH": "6.0.1",
+		"INTEGRATIONS_CORE_BRANCH": "5.22.2",
+		"TRACE_AGENT_BRANCH": "6.0.1",
+		"TRACE_AGENT_ADD_BUILD_VARS": "true",
+		"PROCESS_AGENT_BRANCH": "5.22.1",
```
Comment (sunhay) [inline release.json]: `5.22.2` for process-agent. Just built the binary now.

Comment (sunhay) [reply release.json]: 🙏

**Pull request merged**